### DISCUSSION TITLE
An option to add motion blur

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20955] [Ideas]
It would likely take a lot of work to make something that looks good, but a little bit of motion blur could reduce strain on the eyes and would also make the game feel smoother.

**Comment by ghost** [comment]
where do you want motion blur to be added? in gameplay? in the ui? everywhere?

**Reply by Doksha** [reply]
cursor for standard, and the scrolling objects for the other gamemodes, it doesnt need to be anything extreme, just a couple extra samples.

**Comment by nagi-desuuu** [comment]
Adding motion blur ruins osu in a number of ways, for one, you can miss hit objects due to the extra "flair" you're trying to add.
Not only that, but the triangles mania skin would be impossible with that unnecessary blur!

**Reply by Doksha** [reply]
the triangles skin in mania is basically unreadable in the first place, I can guarantee blur would make it 10x better. Adding a tiny bit of higher sampled blur to allow for an emulation of higher framerates would smooth out the gameplay where it needs it for players who dont have high refresh rate displays. 

blur is a preference, and it may not be for you, but it can help for certain players as well as streamers for better looking gameplay

**Comment by SoffiMeower** [comment]
Just use Reshade if you want motion blur. This would ruin accuracy for a lot of players. Why do you think no other Rhythm Games have it? It makes accuracy near impossible.
